src/MezcalitoUxSearchBundle.php

```diff
@@ -36,6 +36,7 @@ public function configure(DefinitionConfigurator $definition): void
         $rootNode
             ->children()
                 ->scalarNode('default_adapter')->defaultValue('default')->end()
+                ->scalarNode('http_client')->defaultValue('psr18.http_client')->end()
```

Comment:
I think this code is no longer necessary.

Reply:
🙈 but I have removed the code.